teradatalabs/centos6-ssh-oj8/Dockerfile

```diff
@@ -20,9 +20,13 @@ ARG JDK_RPM
 ARG JDK_PATH
 
 # Install Oracle Java and presto-admin dependences (already has python 2.6)
-RUN yum upgrade ca-certificates --disablerepo=epel -y && \
-    yum update -y && \
-    yum install -y wget bzip2 gcc python-devel tar expect && \
+RUN yum install -y \
```

Comment:
No idea, I just saw that it works in other images and ~~let a steam-roller over~~ unified it :)